examples/README.md

```diff
@@ -1,49 +1,48 @@
+---
+render_with_liquid: false
+---
+
 <!-- Copyright Yahoo. Licensed under the terms of the Apache 2.0 license. See LICENSE in the project root. -->
 
 ![Vespa logo](https://vespa.ai/assets/vespa-logo-color.png)
 
-# Vespa Code examples
-
+# Vespa Code And Operational Examples
 
-### [Vespa grouping and facets for organizing results ](part-purchases-demo)
-A sample application demonstrating Vespa grouping and faceting for query time result analytics.
-See [Vespa grouping documentation](https://docs.vespa.ai/en/grouping.html).
+### Vespa grouping and facets for organizing results
+The [part-purchases-demo](part-purchases-demo) app demonstrates Vespa grouping and faceting for query time result analytics. See [Vespa grouping documentation](https://docs.vespa.ai/en/grouping.html).
 
-### [Vespa predicate fields](predicate-fields)
-A sample app which demonstrates how to use Vespa's [predicate field](https://docs.vespa.ai/en/predicate-fields.html) 
-type to implement indexing of document side boolean expressions.
-Boolean document side constraints allows the document to specify which type of queries it can be retrieved for.
-This allows expressing logic like _"this document should only be visible in search for readers in age range 20 to 30"_
-or "This product should only be visible in search during campaign hours".
+### Vespa predicate fields
+The [predicate-fields](predicate-fields) demonstrates how to use Vespa's [predicate field](https://docs.vespa.ai/en/predicate-fields.html) type to implement indexing of document side boolean expressions.
+Boolean document side constraints allows the document to specify which type of queries it can be retrieved for. Predicate fields allow expressing logic like _"this document should only be visible in search for readers in age range 20 to 30"_ or "This product should only be visible in search during campaign hours".
```

Comment:
```suggestion
Boolean document side constraints allows the document to specify which type of queries it can be retrieved for. Predicate fields allow expressing logic like _"this document should only be visible in search for readers in age range 20 to 30"_ or _"this product should only be visible in search during campaign hours"_.
```